docs/requirements.txt

```diff
@@ -1,155 +1,154 @@
-alabaster==0.7.12
-appdirs==1.4.4
-appnope==0.1.2; python_version >= "3.3" and sys_platform == "darwin" or platform_system == "Darwin"
-argon2-cffi==20.1.0
-async-generator==1.10
-atomicwrites==1.4.0; sys_platform == "win32"
-attrs==20.3.0
-babel==2.9.0
-backcall==0.2.0; python_version >= "3.3"
-black==19.10b0
-bleach==3.3.0
-bokeh==2.3.0
-brotli==1.0.9
-cachetools==4.2.1
-certifi==2020.12.5
-cffi==1.14.5
-chardet==4.0.0
-click==7.1.2
-cloudpickle==1.6.0
-codetiming==1.3.0
-colorama==0.4.4; python_version >= "3.3" and sys_platform == "win32" or sys_platform == "win32"
+alabaster==0.7.12; python_version >= "3.5"
```

Comment:
Does it mean we need to specify the python version for each package? How do you know which python version is needed for each package? Did you add them manually?

Reply:
No. This is generated by poetry automatically.